test/k8s/conn/auth/exec_worker_test.exs

```diff
@@ -0,0 +1,119 @@
+defmodule K8s.Conn.Auth.ExecWorkerTest do
+  @moduledoc false
+  use ExUnit.Case, async: true
+
+  alias K8s.Conn.Auth.ExecWorker
+
+  describe "parse_opts/1" do
+    test "parse a comman with nothing else" do
+      config = %{
+        "exec" => %{
+          "apiVersion" => "client.authentication.k8s.io/v1alpha1",
+          "command" => "aws-iam-authenticator"
+        }
+      }
+
+      assert ExecWorker.parse_opts(config) == [
+               command: "aws-iam-authenticator",
+               env: %{},
+               args: []
+             ]
+    end
+
+    test "parse a command with arguments" do
+      config = %{
+        "exec" => %{
+          "apiVersion" => "client.authentication.k8s.io/v1alpha1",
```

Comment:
It probably doesn't matter all that much but v1alpha1 has been removed for quite a while. I don't know that there's a supported version that still has it? Perhaps `v1` would make more sense in that case?